inc/class-cachify.php

```diff
@@ -1108,8 +1128,6 @@ public static function register_flush_cache_hooks() {
 			'create_term' => 10,
 			'delete_term' => 10,
 			'edit_terms' => 10,
-			'user_register' => 10,
-			'edit_user_profile_update' => 10,
```

Comment:
I would not remove those, like @stklcode wrote in https://github.com/pluginkollektiv/cachify/issues/223#issuecomment-820627103, user info can be displayed in many places.

Reply:
Re-added "edit_user_profile_update" for now in decdd88a4d273f4fb5b7e73f8468ba8d26750ede

We can discuss about refining this in future changes.